jobs/scripts/nightly-samba-builds/nightly-samba-builds.sh

```diff
@@ -87,10 +88,28 @@ then
 		echo "Unable to automatically rebase to branch '${ghprbTargetBranch}'. Please rebase your PR!"
 		exit 1
 	fi
+
+	proceed=0
+
+	read -r -d '' -a FILES_CHANGED < <(git diff --name-only origin/samba-build)
```

Comment:
Instead of hardcoding `samba-build`, you may want to use `origin/${BUILD_GIT_BRANCH}` as it it cloned above (or maybe `ghprbTargetBranch`?)

Reply:
Err... I did my testing with hard-coded branch name and copy-pasted. Updated now. Good catch :-)